spec/jobs/address/check_validity_job_spec.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2012-2020, CVP Schweiz. This file is part of
+#  hitobito_cvp and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_cvp.
+
+require 'spec_helper'
+
+describe Address::CheckValidityJob do
+
+  let(:job) { Address::CheckValidityJob.new }
+  let(:validator) { Contactable::AddressValidator.new }
+
+  before do
+    expect(Contactable::AddressValidator).to receive(:new).and_return(validator)
+  end
+
+  it 'sends email if invalid people are found and mail address is defined' do
+    Settings.addresses.validity_job_notification_emails = ['mail@example.com']
```

Comment:
Die Settings werden email für die suite geladen, nicht pro test run, sprich du überschreibst hier eine globale Variable ohne sie zurückzusetzen. 
Hier besser mit `allow(Settings.addresses).to receieve(..).and_return(..)` arbeiten